store/contentSchemas.json

```diff
@@ -26,6 +26,11 @@
           "$ref": "app:vtex.native-types#/definitions/url",
           "default": ""
         },
+        "callToActionLinkTarget": {
+          "title": "admin/editor.info-card.callToActionLinkTarget.title",
+          "enum": ["_self", "_blank"],
+          "enumNames": ["Same tab", "New tab"]
```

Comment:
Make sure to use the `messages` for these `enumNames` too, so that they are translatable, just like you did for the title property.

Reply:
@victorhmp não funcionou :( 
ele renderizou o "admin/editor.info-card..." ao invés de renderizar o texto que estava dentro. Você tem alguma ideia do porque??

Reply:
Aaah sim! Não tinha percebido que isso aq estava no `contentSchemas.json`, desculpa 😅.
Nesse caso, acho melhor você mover essa declaração aqui pra dentro do componente, e fazer algo parecido com isso aq: https://github.com/vtex-apps/store-components/blob/master/react/components/InfoCard/index.js#L298-L306.
Melhor ainda pq vc usa as funções `getEnumValues` e `getEnumNames` e elas já vão retornar os valores corretos pra você :)

Reply:
vou dar uma olhada!! obrigadaa

Reply:
@victorhmp eu tentei fazer essa alteração, mas não funcionou :( ta faltando alguma coisa shuahsua você poderia me ajudar??

Reply:
Claro! Pelo erro que está dando, acho que se você remover a declaração da prop do `contentSchemas.json` ele já vai funcionar :)
Outra coisa que notei que está faltando são as `messages` no arquivo `en.json`.

Reply:
@victorhmp mas então como que eu faço pra essa prop aparecer no site editor? sem ter nada no contentSchemas.json

Reply:
Se a prop estiver no objeto `.schema` do componente React, vai aparecer no Site Editor do mesmo jeito. Acabei de perceber só um detalhe: tira a propriedade `isLayout` da definição pra prop que você está adicionando! Ou colocar `isLayout: false`.

Reply:
@victorhmp tirei, mas o objeto esta no `.schema` mas não está aparecendo no site-editor :(

Reply:
🤔 Acho que eu saquei o motivo... no `interfaces.json` estamos definindo que o `content` do `InfoCard` é definido assim:

```json
"content": {
      "$ref": "#/definitions/InfoCard"
    }
```

O que isso quer dizer para o CMS é: "o conteúdo editável do InfoCard está em no `definitions.InfoCard` do `contentSchemas.json`". Então ele só está ignorando as props definidas em `InfoCard.schema` :(

Desculpa não ter percebido isso antes! São muitos componentes para lembrar desses detalhes 😅 . Enfim, acho que a solução mais rápida agora, é fazer isso pelo `contentSchemas.json` mesmo. Deve ser algo assim:

```json
"callToActionLinkTarget": {
    "default": "_self",
    "type": "string",
    "title": "admin/editor.info-card.callToActionLinkTarget.title",
     "enum": [
         "_blank",
          "_self",
          "_parent",
          "_top"
      ],
      "enumNames": [
           "admin/editor.info-card.callToActionLinkTarget.blank",
           "admin/editor.info-card.callToActionLinkTarget.self",
           "admin/editor.info-card.callToActionLinkTarget.parent",
            "admin/editor.info-card.callToActionLinkTarget.top"
        ]
},
```

Denovo, desculpa pela confusão, não tinha percebido isso antes 😢